pkgs/servers/kanidm/default.nix

```diff
@@ -24,8 +24,8 @@ rustPlatform.buildRustPackage rec {
   src = fetchFromGitHub {
```

Comment:
If we want to keep this, we should append ``-unstable-XX-XX-XXXX`` to the version variable and fill in the Xs with the date of hte commit in iso format.

Reply:
It is weird to call it unstable since it comes from the backports branch for version 1.1.0-rc.16 of kanidm, not the master one. Previous commits on kanidm didn't specified this suffix (https://github.com/NixOS/nixpkgs/pull/264658/commits/650e5169319788e2a06468585c59aaf1a355166f). May be it is better to put something like `rev-2024-02-28`

Reply:
What do you think ?

Reply:
I don’t care that much about naming things. The way we did it previously (adding a comment that this is from a release branch) is fine with me.
We shouldn’t mark it as `unstable` if it’s from a release branch.

Reply:
Done